### PR TITLE
Add some comment of CrossMapNormalFunc

### PR DESCRIPTION
**Pull request by hedaoyuan**
According to the comment of @emailweixu .
Each Function needs a comment, and
1. Do not need to read the code, to know what Function is doing
2. Do not need to read the code, to know the meaning of the Function parameter, and how to use this Function;

Normalization is a complex formula, I am not sure if it's clear. Please help to review this pr.

### CODE REVIEW

File: paddle/function/CrossMapNormalOp.cpp [hunk 1]
```diff
@@ -112,11 +112,31 @@ void CrossMapNormalGrad<DEVICE_TYPE_CPU>(real* inputsGrad,
...
+ * "ImageNet Classification with Deep Convolutional Neural Networks".
+ *
+ * The original formula is:
+ *
+ *                                 Input(x, y)
+ * Output(x, y) = ------------------------------------------------
+ *                       alpha   /min(F, f-[N/2] + N)
+ *                  (1 + ----- * |    (Input(x, y))^2 ) ^ (beta)
+ *                         N     /max(0, f-[N/2])
+ *
```
Comment (qingqing01) [inline paddle/function/CrossMapNormalOp.cpp]: 1. 看这个公式，无法看出 across channel (or maps)
2. Input/Output是4D的tensor，注释里面如果忽略第一维的话， Input(x, y) 是不是应该写作  Input(c, x, y) (或者 Input[, , , ]) ?
3. 求和符号， 直接用 sum 会不会更清晰些， 或者

```
   --
   \
   /
   --
```
4. paper的公式里是 +k，而不是+1。
5. alpha不用除N吧？

另外，如果比较复杂的公式，如果拆解开来写多行，会不会好些一些

File: paddle/function/CrossMapNormalOp.cpp [hunk 2]
```diff
@@ -112,11 +112,31 @@ void CrossMapNormalGrad<DEVICE_TYPE_CPU>(real* inputsGrad,
...
+ * Argument in the Function:
+ * \param size_      represent N
+ * \param scale_     represent alpha / N
```
Comment (qingqing01) [inline paddle/function/CrossMapNormalOp.cpp]: paper的公式，直接是 alpha，看代码实现了scale_等价公式里的alpha吧？ 不是alpha/N吧？ paper: https://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf

File: paddle/function/CrossMapNormalOp.cpp (hunk 1 above)
Comment (hedaoyuan) [inline paddle/function/CrossMapNormalOp.cpp]: Done.

File: paddle/function/CrossMapNormalOp.cpp (hunk 2 above)
Comment (hedaoyuan) [inline paddle/function/CrossMapNormalOp.cpp]: Done.

**Pull request merged**